contracts/CreateAccount.sol

```diff
@@ -160,22 +160,38 @@ contract CreateAccount is FraudProofHelpers {
         // Assuming Reddit have run rollup.sol::createPublickeys
         ValidatePubkeyAvailability(_accountsRoot, _to_pda_proof, _tx.toIndex);
 
-        // accountProofs.to.accountIP.account should to be a zero account
-        bool result = ValidateZeroAccount(accountProofs.to.accountIP.account);
-        if (result == false) {
-            return ("", "", "", 0, false);
+        // Validate we are creating on a zero account
```

Comment:
We need a signature check too I think?

Reply:
This requires passing "from" PDA proof to processTx, and we'll have 2 PDA proofs (from and to) in the processTx. If we are doing BLS and validate this at the batch level, I would prefer to do it in the next PR.